examples/tensorflow_converter/squeezenet_benchmark.py

```diff
@@ -0,0 +1,536 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org_licenses_LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Timing benchmark for SqueezeNet inference.
+
+To run, use:
+  bazel run -c opt --config=cuda \
+      third_party_tensorflow_models_image_squeezenet:squeezenet_benchmark
```

Comment:
Thanks for notification, underscores is wrong. I use batch replacement on some operation name from original caffe prototxt for SqueezeNet which causes such mistake.